shared/common-adapters/avatar.render.desktop.js

```diff
@@ -33,46 +34,49 @@ type State = {
 const backgroundOffset = 1
 class Background extends PureComponent<void, {loaded: boolean, loadingColor: ?string}, void> {
   render() {
-    const {loaded, loadingColor} = this.props
-    return (
-      <div
-        style={{
-          backgroundColor: loaded ? globalColors.white : loadingColor || globalColors.lightGrey,
-          borderRadius: '50%',
-          bottom: backgroundOffset,
-          left: backgroundOffset,
-          position: 'absolute',
-          right: backgroundOffset,
-          top: backgroundOffset,
-        }}
-      />
+    const Div = glamorous.div(
```

Comment:
and then i'd have to make sep. props or use {...style} to pass it all down right? didn't seem better at the time